support/BRFileService.c

```diff
@@ -152,9 +312,17 @@ fileServiceRelease (BRFileService fs) {
     size_t typesCount = array_count(fs->entityTypes);
     for (size_t index = 0; index < typesCount; index++)
         fileServiceEntityTypeRelease (&fs->entityTypes[index]);
+    array_free(fs->entityTypes);
+
+
+    free (fs->network);
+    free (fs->currency);
+    free (fs->sdbPath);
+    sqlite3_finalize(fs->sdbInsertStmt);
```

Comment:
As you mentioned, not all statements are released.

Reply:
yes.